src/main/kotlin/org/kamiblue/client/module/AbstractModule.kt

```diff
@@ -33,9 +35,11 @@ abstract class AbstractModule(
     private val enabled = BooleanSetting("Enabled", false, { false }).also(::addSetting)
     private val visible = BooleanSetting("Visible", showOnArray).also(::addSetting)
     private val default = BooleanSetting("Default", false, { settingList.isNotEmpty() }).also(::addSetting)
+    val priorityForGui = IntegerSetting("Priority In GUI", 0, IntRange(0, 1000), 1, { ClickGUI.sortBy.value == ClickGUI.SortByOptions.CUSTOM }).also(::addSetting)
+    val clicks = IntegerSetting("Clicks", 0, IntRange(0, Int.MAX_VALUE), 1, { false }).also(::addSetting) // Not nice, however easiest way to save it.
```

Comment:
Use `<min>..<max>` instead if you want to construct an IntRange
```suggestion
    val priorityForGui = IntegerSetting("Priority In GUI", 0, 0..1000, 1, { ClickGUI.sortBy.value == ClickGUI.SortByOptions.CUSTOM }).also(::addSetting)
    val clicks = IntegerSetting("Clicks", 0, 0..Int.MAX_VALUE, 1, { false }).also(::addSetting) // Not nice, however easiest way to save it.
```